resources/language/resource.language.de_de/strings.po

```diff
@@ -1762,44 +1761,44 @@ msgstr "Langsame Verbindung"
-msgstr "Mit wackeliger/langsamer Verbindung benutzen, z.B. im Hotel. Passt die Oberfläche visuell an um auf Media Auffrischungen zu warten und füllt den Kodi Puffer vor dem Abspielen. Setzt automatisch Kodi Cache readfactor=20, benötigt Kodi Neustart."
+msgstr "Bei langsamer Verbindung benutzen, z.B. im Hotel. Passt die Oberfläche visuell an um auf Media Auffrischungen zu warten und füllt den Kodi Puffer vor dem Abspielen. Setzt automatisch Kodi Cache readfactor=20, benötigt einen Kodi Neustart."
 
 msgctxt "#32917"
 msgid "Couldn't fill buffer in time ({}s)"
-msgstr "Konnte den Puffer nicht schnell genug füllen ({} s)"
+msgstr "Konnte den Puffer nicht schnell genug füllen ({} Sek.)"
 
 msgctxt "#32918"
 msgid "Buffer wait timeout (seconds)"
 msgstr "Puffer-Wartezeit (Sekunden)"
 
 msgctxt "#32919"
 msgid "When slow connection is enabled in the addon, wait this long for the buffer to fill. Default: 120 s"
-msgstr "Wenn Langsame Verbindung im Addon aktiviert ist, warte so lange bis der Puffer gefüllt ist. Default: 120 s"
+msgstr "Wenn Langsame Verbindung im Addon aktiviert ist, warte so lange bis der Puffer gefüllt ist. Voreinstellung: 120 Sek."
 
 msgctxt "#32920"
 msgid "Insufficient buffer wait (seconds)"
 msgstr "Ungenügender Puffer Wartezeit (Sekunden)"
 
 msgctxt "#32921"
 msgid "When slow connection is enabled in the addon and the configured cache/buffer isn't big enough for us to determine its fill state, wait this long when starting playback. Default: 10 s"
-msgstr "Wenn Langsame Verbindung im Addon aktiviert und der konfigurierte Puffer/Cache nicht groß genug ist, damit wir dessen Füllstand ermitteln können, warte so lange vor dem Abspielen. Default: 10 s"
+msgstr "Wenn \"Langsame Verbindung\" im Addon aktiviert ist und der konfigurierte Puffer/Cache nicht groß genug ist, um seinen Füllstand zu ermitteln, wird so lange gewartet, bevor die Wiedergabe gestartet wird. Voreinstellung: 10 Sek."
 
 msgctxt "#32922"
 msgid "Kodi Cache Readfactor"
 msgstr "Kodi Puffer Readfactor"
 
 msgctxt "#32923"
 msgid "Sets the Kodi cache readfactor value. Default: 4, recommended: 4-10. With \"Slow connection\" enabled this will be set to 20, as otherwise the cache doesn't fill fast/aggressively enough."
-msgstr "Setzt den Kodi Cache/Puffer readfactor Wert. Default: 4, empfohlen: 4-10. Bei aktivem \"Langsame Verbindung\" wird dies automatisch auf 20 gesetzt, da ansonsten der Cache nicht schnell/agressiv genug gefüllt wird."
+msgstr "Setzt den Kodi Cache/Puffer readfactor Wert. Standard: 4, Empfohlen: 4-10. Bei aktivem \"Langsame Verbindung\" wird dies automatisch auf 20 gesetzt, da ansonsten der Cache nicht schnell/agressiv genug gefüllt wird."
 
 msgctxt "#32924"
 msgid "Minimize"
 msgstr "Minimieren"
 
 msgctxt "#32925"
 msgid "Playback Settings"
-msgstr "Abspieleinstell."
+msgstr "Wiedergabe-Einstellungen"
```

Comment:
This will overflow in the GUI and show "Wiedergabe-Einstellu..."

Reply:
Resolved!